File: geometry/distance_formula/ruby/distance_formula.rb
include Math
require 'cmath'

def distance_formula(point_1, point_2)

  distance = CMath.sqrt((point_1[0]-point_2[0])^2 + (point_1[1]-point_2[1])^2)
  distance

end


p1 = [10, 0]
p2 = [10, 0]

puts "distance between point_1 and point_2 is  #{distance_formula(p1, p2)}"

File: geometry/distance_formula/php/distance_formula.php
<?PHP

function euclidianDistance($p1,$p2)
{
    $dX = $p1.x - $p2.x;
    $dY = $p1.y - $p2.y;
    $distance = sqrt($dX*$dX+$dY*$dY);

    return $distance;
};

?>
